### PR TITLE
Add start/stop level change report event handling to SwitchMultilevel

### PR DESCRIPTION
**Pull request by ericyeargan**
This change allows reception of dim up/down events from dimmer controller paddles. Unfortunately, I also changed the message type handling from an if/else ladder to a switch which makes the diff difficult to parse. Let me know if I can help clarify.

### CODE REVIEW

File: cpp/src/command_classes/SwitchMultilevel.cpp [hunk 1]
```diff
@@ -166,70 +168,162 @@ bool SwitchMultilevel::HandleMsg
...
-	}
 
-	if( SwitchMultilevelCmd_SupportedReport == (SwitchMultilevelCmd)_data[0] )
-	{
-		uint8 switchType1 = _data[1] & 0x1f;
-		uint8 switchType2 = _data[2] & 0x1f;
-		uint8 switchtype1label = switchType1;
-		uint8 switchtype2label = switchType2;
-		if (switchtype1label > 7) /* size of c_switchLabelsPos, c_switchLabelsNeg */
+		case SwitchMultilevelCmd_Set:
```
Comment (Fishwaldo) [inline cpp/src/command_classes/SwitchMultilevel.cpp]: Why did you implement Set? OZW is not a Device. We dont expose the SwitchMultiLevel CC in our NIF so this code will never be executed/Called.

Please Delete this portion.

**Comment by Fishwaldo** [comment]
Please see my comments, and drop the Set Handling. Thats "dead" code.

File: cpp/src/command_classes/SwitchMultilevel.cpp (hunk 1 above)
Comment (ericyeargan) [inline cpp/src/command_classes/SwitchMultilevel.cpp]: I believe I added that while trying to get the bright/dim received events to work. Will remove...

**Comment by Fishwaldo** [comment]
Hi,
Apologies, I just cleaned up the repo, removing old object files from way back when. This has resulted in PR's being messed up. Could you please redo the PR after pulling a new fork (the repo should be much smaller now though)

Thanks

**Comment by ericyeargan** [comment]
@Fishwaldo, no problem - new PR submitted...

**Pull request closed without merging**